amp-conformance/CMakeLists.txt

```diff
@@ -1,4 +1,7 @@
-include_directories(${CMAKE_CURRENT_SOURCE_DIR}/amp_test_lib/inc ${LIBCXX_HEADER})
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/amp_test_lib/inc)
+if (USE_LIBCXX)
+  include_directories(${LIBCXX_HEADER})
```

Comment:
If this include is necessary to use the amp libraries, then the include should be added to the libraries' target, and not here. Otherwise, the user will need to add this line every time they link against the libraries.